packages/@aws-cdk/cloud-assembly-schema/lib/cloud-assembly/metadata-schema.ts

```diff
@@ -312,6 +312,11 @@ export enum ArtifactMetadataEntryType {
    * Represents tags of a stack.
    */
   STACK_TAGS = 'aws:cdk:stack-tags',
+
+  /**
+   * Whether the resource should be skipped during refactoring.
+   */
+  SKIP_REFACTOR = 'aws:cdk:skip-refactor',
```

Comment:
Should this be called exclude now? Or

```suggestion
  SKIP_REFACTOR = 'aws:cdk:no-refactor',
```